query/src/main/java/org/infinispan/query/impl/PartitionHandlingSupport.java

```diff
@@ -0,0 +1,39 @@
+package org.infinispan.query.impl;
+
+import org.infinispan.AdvancedCache;
+import org.infinispan.configuration.cache.ClusteringConfiguration;
+import org.infinispan.partitionhandling.AvailabilityMode;
+import org.infinispan.partitionhandling.PartitionHandling;
+import org.infinispan.query.logging.Log;
+import org.infinispan.util.logging.LogFactory;
+
+/**
+ * @since 9.3
+ */
+public final class PartitionHandlingSupport {
+
+   private static final Log LOGGER = LogFactory.getLog(CacheQueryImpl.class, Log.class);
```

Comment:
Wrong logger

Reply:
Dan, I already said it's the wrong logger. So I guess nobody sees my comments :(